src/aura/HH_AutoComplete/HH_AutoComplete.cmp

```diff
@@ -29,13 +29,8 @@
 
         <aura:set attribute="listFooter">
             <div class="slds-lookup__item">
-                <ui:button class="slds-button" press="{!c.handleNewContact}">
-                    <c:svg ariaHidden="true"
-                           class="slds-icon slds-icon-text-default slds-icon--small"
-                           xlinkHref="{!$Resource.sfdo203 + '/assets/icons/utility-sprite/svg/symbols.svg#add'}"
-                           />
-                    {!$Label.npo02.NewContact}
-                </ui:button>
+                <lightning:button variant="bare" label="{!$Label.npo02.NewContact}"
+                    iconName="utility:add" iconPosition="left" onclick="{!c.handleNewContact}" />
             </div>
```

Comment:
for some reason with your changes, the loadingIndicator spinner now displays in the middle of the main window, rather than just within the autocomplete component.  I verified this isn't happening in my other DE org on Winter '17 without your changes.  I suspect its due to moving from our scoped SLDS to the system one.  Note sure what we need to change to get the spinner in the correct location.